contrib/win32/win32compat/console.c

```diff
@@ -227,8 +227,12 @@ ConEnterRawMode()
 void 
 ConExitRawMode()
 {
-	SetConsoleMode(GetStdHandle(STD_INPUT_HANDLE), stdin_dwSavedAttributes);
-	SetConsoleMode(GetStdHandle(STD_OUTPUT_HANDLE), stdout_dwSavedAttributes);
+	if (0 == in_raw_mode) {
```

Comment:
once in_raw_mode is set.. it is never reset back to 0.. Please add this at the end of this method.

Reply:
I am merging this PR. I will fix this